src/components/useMeeting.ts

```diff
@@ -0,0 +1,110 @@
+import {
```

Comment:
I like the fact that we can put all the logic into one file. On the other hand this will probably make the file quite long if add all the functions we need (updating, saving, deleting different fields). What I cannot find here tho is where do we update the backend ?

Reply:
@attilakiss323 check like 38-45. State management and async request handling are 2 different topics, but I added a small text explaining how we could deal with it in case of failure